README.md

```diff
@@ -53,6 +53,30 @@ username: nil,
 password: nil,
 ```
 
+### Creating collections (migrations)
+
+The ArangoDB adapter handles table and index create statements. Since ArangoDB
+is schemaless, you do not need to specify fields in your migrations.  
+ArangoDB does have some special `options` when creating collections:
+- "edge": the collection type is [edge](https://docs.arangodb.com/devel/Manual/Graphs/Edges/)
+- "uuid": the primary key type is UUID (introduced with ArangoDB 3.4)
+- "autoincrement": the primary key is autoincrementing
+
+To define collections for storing assemblies containing assemblies:
+```elixir
+defmodule Db.Repo.Migrations.CreateAssemblies do
+  use Ecto.Migration
+
+  def change do
+    create(table(:assemblies, options: "uuid"))
+    create(table(:contains, options: "edge"))
```

Comment:
How about using a Keyword list for the options? Possible values:
* collection_type: document | edge
* key_generator: traditional | autoincrement | padded | uuid
This would be more idiomatic and allow us to add more options in the future.

Reply:
Right, I knew there was a reason I didn't implement it this way in the first place - I just couldn't remember why. Too bad, but in this case the current solution LGTM.